console/console-init/ui/src/modules/project/ProjectPage.tsx

```diff
@@ -149,6 +158,49 @@ export default function ProjectPage() {
       }
     }
   };
+  const handleOnChangeEnable = (projects: IProject[], toEnable: boolean) => {
```

Comment:
can you please try function name onChangeStatus or handleStatus and change param name toEnable ==>status